### PR TITLE
Add the empty graph

### PR DESCRIPTION
**Pull request by aneeshdurg**
I think eventually it should be possible for csv-import to just create this graph, but until then, I was encouraged to drop this here.

### CODE REVIEW

File: rdg_datasets/empty_graph/storage_format_version_6/part_vers00000000000000000001_rdg_node00000 [hunk 1]
```diff
@@ -0,0 +1 @@
...odes_0-23jWTL2DMULR"],["host_to_owned_global_node_ids","host_to_owned_global_node_ids-hHqPT6iy7kjx"],["host_to_owned_global_edge_ids","host_to_owned_global_edge_ids-Q4m0LaiObIVM"],["local_to_user_id","local_to_user_id-EOOFpx08vFat"],["local_to_global_id","local_to_global_id-xT6MW7VcRDzN"]],"kg.v1.part_property_meta":{"cartesian_grid":[0,0],"is_incoming_edge_cut":false,"is_outgoing_edge_cut":false,"magic":1266713220,"max_global_node_id":18446744073709551615,"num_edges":0,"num_global_edges":0,"num_global_nodes":0,"num_nodes":0,"num_owned":0,"policy_id":0,"transposed":false},"kg.v1.partition_topology_metadata":{"kg.v1.partition_topology_metadata_entries":[{"edge_condensed_type_id_map_present":false,"edge_condensed_type_id_map_size":0,"edge_index_to_property_index_map_present":false,"edge_sort_state":"kAny","node_condensed_type_id_map_present":false,"node_condensed_type_id_map_size":0,"node_index_to_property_index_map_present":false,"node_sort_state":"kAny","num_edges":0,"num_nodes":0,"path":"topology-dyqVp74lR46R","topology_state":"kCSR","transpose_state":"kNo"}],"kg.v1.partition_topology_metadata_entries_size":1},"kg.v1.storage_format_version":5,"kg.v1.unstable_storage_format":false}
```
Comment (e-mcginnis) [inline rdg_datasets/empty_graph/storage_format_version_6/part_vers00000000000000000001_rdg_node00000:1]: this rdg is `kg.v1.storage_format_version":5`, please put it in a `storage_format_version_5` directory

apologies, v6 is just making its way into the codebase as we speak

**Review by e-mcginnis — CHANGES_REQUESTED** [review]
Please add a script to import this from an empty csv automatically for upreving the storage format version, doing this manually every time we bump the storage format version wastes a lot of time fast. 

 I can help you out with writing this is you'd like

EDIT: you can take a look at this import script for the smiles_small rdg https://github.com/KatanaGraph/test-datasets/blob/master/rdg_datasets/smiles_small/import.py

**Comment by aneeshdurg** [comment]
So I tried writing an import script, but it didn't seem to like importing from an empty file. Right now the only tool that really supports generating valid empty-graphs is csv-import. Maybe I can write a generate script instead?

**Comment by e-mcginnis** [comment]
> Right now the only tool that really supports generating valid empty-graphs is csv-import

The backend of the import tooling is csv-import
https://github.com/KatanaGraph/test-datasets/blob/219aaa51568c45e4c9599a160ee2b95265b06310/libuprev/rdg_import.py#L49
so something else must be up.
If you want to share the code & error with me, I can take a look?

**Review by e-mcginnis — APPROVED** [review]

**Pull request merged**